UPGRADE-PHP-TO-TWIG-TEMPLATES.md

```diff
@@ -93,6 +94,14 @@ Becomes
 
     ```
 - Regarding the `default` filter: pay special attention to the [documented use case](https://twig.symfony.com/doc/3.x/filters/default.html) where false is treated as an empty value. Consider using `?? true` instead of `|default(true)` when using the default filter on a variable that could already have a boolean value set.
-  
+- You can use text replacement with the `{% trans %}` tag as it is built in:
+    ```
+    {% trans with {'%code%': status_code} %}message{% endtrans %}
+    ```
+    Alternatively, you can use the `|trans` filter instead. This is a bit more compact and and handy, since you can chain multiple filters. Combining translation with text replacement becomes:
+
+    ````
+    {{message | trans | replace({"%code%": status_code})}}
```

Comment:
The equivalent to `{% trans with {'%code%': status_code} %}message{% endtrans %}` is `{{ 'message'|trans({'%code': status_code}) }}`

A better example of chaining filters would be `{{ message|striptags|title }}`. Another example is the `filter` and `length` filters. For example, `{% if data|filter(v => v > 42)|length > 0 %}`.

Reply:
Right, I didn't put it together that the trans filter had its own built-in replace functionality even though the tag version had it.

Reply:
@LordRembo is this resolved?